packages/ingest-mongodb-public/package.json

```diff
@@ -15,6 +15,7 @@
     "watch": "tsc -b -w",
     "release": "release-it",
     "ingest:all": "ingest all --config ./build/config.js",
```

Comment:
interestingly, this `ingest:all` command works locally AND in a local docker image using the same ingest-service.dockerfile. but the `ingest` binary doesn't work in the k8s-deployed docker image for reasons i do not understand. will ask kanopy team if they can explain

Reply:
but in the meantime, we can merge this as-is i think